wremnants/CardTool.py

```diff
@@ -715,7 +731,774 @@ def writeOutput(self, args=None, xnorm=False, forceNonzero=True, check_systs=Tru
         if self.skipHist:
             logger.info("Histograms will not be written because 'skipHist' flag is set to True")
         self.writeCard()
-        logger.info(f"Output stored in {self.outfolder}")
+
+    def writeHDF5Output(self, args=None, xnorm=False, theoryFit=False,
+        simultaneousABCD=False,
+        forceNonzero=True, check_systs=False, allowNegativeExpectation=False, 
+        sparse=False, dtype="float64", chunkSize=4*1024**2,
+        logkepsilon=math.log(1e-3), #numerical cutoff in case of zeros in systematic variations
```

Comment:
Probably a lot of these should be class variables that can be set from the card_tool object